core/src/main/scala/com/microsoft/azure/synapse/ml/logging/BasicLogging.scala

```diff
@@ -52,10 +52,6 @@ trait BasicLogging extends Logging {
     logVerb("transform", f)
   }
 
-  def logPredict[T](f: => T): T = {
```

Comment:
why this?  just curious

Reply:
It was used in a weird place (every prediction row in LGBM) which caused some serious slowdowns and console spam